requirements.txt

```diff
@@ -10,7 +10,7 @@
 # scripts/convert-requirements-to-conda-yml.py as though it can only be found
 # on pip.
 
-GDAL>=3.4.2
+GDAL>=3.4.2,<3.8.5
```

Comment:
Is this redundant to the constraints file?

Reply:
Sure is!  Patched in 784735bcf